test/libsolidity/syntaxTests/inlineAssembly/if_statement_invalid_2.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f() public pure {
+        assembly {
+            if mstore(1, 1) {}
+        }
+    }
+}
+// ----
+// TypeError 3950: (78-90): Expected expression to evaluate to one value, but got 0 values instead.
```

Comment:
Why didn't you turn this into a yul syntax test?

Reply:
@chriseth because it's a TypeError, and not actually a syntax test. That's been my question on Wednesday how to treat non-syntax tests. I -now- however think they should all be moved.